README.md

```diff
@@ -127,17 +126,17 @@ good installation path if you are just testing or doing development on the plugi
 
 For more permanent installs of the nexus-repository-p2 plugin, follow these instructions:
 
-* Copy the bundle (nexus-repository-p2-0.0.2.jar) into <nexus_dir>/deploy
+* Copy the bundle (nexus-repository-p2-1.0.0.jar) into <nexus_dir>/deploy
```

Comment:
do we need this one sentence? It will not work from deploy. because plugin already installed

Reply:
FYI, I assumed all the plugin stuff was for older versions (like if someone still wanted to use 0.0.2).  That being said, updating to 1.0.0 does make this slightly confusing...and maybe I'm just wrong :D

Reply:
@aornatovskyy check line 100 it says that current words only for old versions